setup.cfg

```diff
@@ -75,8 +75,7 @@ filterwarnings=
     ignore: 'cgi' is deprecated and slated for removal in Python 3\.13:DeprecationWarning
 
 [coverage:run]
-omit = venv/*
-include = uvicorn/*, tests/*
+source_pkgs = uvicorn, tests
```

Comment:
This should change anyway because we are missing files in the coverage report :man_facepalming: